sdk/src/signature.rs

```diff
@@ -167,15 +179,23 @@ where
     }
 }
 
-#[derive(Debug, Default)]
+impl<T> From<T> for Box<dyn KeypairUtil>
+where
+    T: KeypairUtil + 'static,
```

Comment:
Why `'static`?

Reply:
@t-nelson

Reply:
Oh! `+ 'static` isn't necessary here.  I was following the compiler at one point and later figured out it was being overly specific.  I must've missed removing this one